### ISSUE TITLE
Use parenthesis in  (x &amp; 1) == 1

### ISSUE THREAD

**Issue by axkr** [spotbugs/spotbugs#1995]
https://github.com/spotbugs/spotbugs/blob/818b14cee799caffc9cc0ceefa560e24b82921ce/spotbugs/etc/messages.xml#L7761

should be changed to 

```
for oddness, consider using (x &amp; 1) == 1, or (x % 2) != 0.
```

**Comment by 4sujittiwari** [comment]
@KengoTODA @axkr I have raised a PR to update the documentation with parenthesis. 

PR [2002](https://github.com/spotbugs/spotbugs/pull/2002)

**Comment by KengoTODA** [comment]
closed by #2002
